dist/tools/randhex/randhex.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+bits="${1:-64}"
+if [ $((bits % 2)) -ne 0 ]; then
+	echo "${0##*/}: Amount of bits must be even" 1>&2
+	exit 1
+fi
+
+randhex="$(tr -dc 'A-F0-9' < /dev/urandom | head -c $((bits / 4)))"
```

Comment:
The generation of random numbers in this script is actually insufficient. This has to be rewritten entirely.